packages/web/components/token-details/token-details.tsx

```diff
@@ -118,11 +120,7 @@ const TokenDetails = ({
     }
   }, [coinGeckoId]);
 
-  const name = useMemo(() => {
-    if (details) {
-      return details.name;
-    }
-
+  const currency = useMemo(() => {
```

Comment:
I know this is unrelated to the PR, but we have a handy `getAsset` tRPC function that can take any denom (base IBC denom or ticker symbol) and return currency info. This may be more scalable than importing the chainlist into the page.

I'd appreciate it if you could open a PR and use that in place of importing the chainlist anywhere the chainlist is used. LMK if you have any other questions. You can see an example of `getAsset` tRPC query in the `useSwap` hook.

Reply:
I'll open a new PR for that, thank you!